cmd/flow_capture.go

```diff
@@ -397,6 +345,30 @@ func updateTable() {
 	}
 }
 
+func cycleOption(selection []string, exclusiveOptions []string, options []string, incr int) []string {
```

Comment:
also maybe a comment such as (if I understand correctly):
> Cycles through options, with exclusive options first, then non-exclusive options, and last the full list of options all at once.

Reply:
@jpinsonneau did you see my comments here ?

I forgot you were doing this and ended up something similar here https://github.com/netobserv/network-observability-cli/pull/137/files#diff-a86c8e951a1155a451b2d254dfecbb2d151455820c7954bc9795a96c8379b2a5R16-R24 , which I'll remove this you were here first :-)
But I have the feeling it can be simplified when using more structured data... at the end of the day, cycling is just incrementing or decrementing an index that could be persisted in memory rather than having to re-compute it by lookups and matching

Reply:
Created https://issues.redhat.com/browse/NETOBSERV-2016 to address that in a followup :wink: 
Thanks !